### ISSUE TITLE
Long term memory impairment

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6769]
**Preferred term label:**
Long term memory impairment

**Synonyms**
Long term memory loss

**Definition (free text, please give PubMed ID)**
A deficit in the ability to retrieve information from long-term memory, which can be defined as a seemingly unlimited capacity to store memories can last years and relate to the performance of actions or skills (i.e., procedural memories, knowing how) and memories of facts, rules, concepts, and events (i.e., declarative memories, knowing that).
PMID:31424720
PMID:18394484

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0002354

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
